index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <main>
+      <header>
+        <h1 id="title">Complete Below form</h1>
+        <p id="description">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Cumque,
+          error!
+        </p>
+      </header>
+
+      <form id="survey-form">
+        <label id="name-label">Fullname: </label><br />
+        <input type="text" id="name" placeholder="enter fullname" required />
+        <br />
+        <label id="email-label">Email:</label><br />
+        <input type="email" id="email" placeholder="enter email" required />
+        <br />
+
+        <label id="number-label">Number: </label><br />
+        <input
+          type="number"
+          id="number"
+          placeholder="enter any number"
+          min="0"
+          max="100"
+          required
+        />
+
+        <br /><label>Select continent: </label><br />
+        <select name="name" id="dropdown">
```

Comment:
To avoid the use of br tag, I'll recommend using something like display flex and flex-direction: column to make the form Items stack on each other.

Reply:
- [ ] I would suggest explaining "why" you are giving a recommendation, in this situation, you could explain why avoiding the use of the `<br>` tag is a better practice. Remember that when you explain "why", the user will feel that is learning something new and it will make the review more friendly :wink:
- [ ] [OPTIONAL] I think you could try adding some emojis to your comments to make the communication more human :smiley: